wagtailcaptcha/models.py

```diff
@@ -18,14 +18,10 @@ def __init__(self, *args, **kwargs):
         self.form_builder = WagtailCaptchaFormBuilder
 
     def process_form_submission(self, form):
-        super(AbstractEmailForm, self).process_form_submission(form)
-
-        if self.to_address:
-            content = ''
-            for x in form.fields.items():
-                if not isinstance(x[1], ReCaptchaField):  # exclude ReCaptchaField from notification
-                    content += '\n' + x[1].label + ': ' + text_type(form.data.get(x[0]))
-            send_mail(self.subject, content, [self.to_address], self.from_address,)
+        if WagtailCaptchaFormBuilder.CAPTCHA_FIELD_NAME in form.fields.keys():
+            form.fields.pop(WagtailCaptchaFormBuilder.CAPTCHA_FIELD_NAME)
+
+        return super(WagtailCaptchaEmailForm, self).process_form_submission(form)
```

Comment:
I don't think we really care about the captcha being saved or not. I created the discussion about saving/not saving the captcha code on the database because I thought the refactoring would change the current behaviour but it doesn't. Therefore the to_address not being correctly handle is solved and the behaviour is preserved, I think we are good to go, but I would like your opinion as well. 

In my opinion the discussion about saving/not saving the captcha on the database should be done in a different Issue/PR.

PS: Yes, I'm also curious about why wagtail used fields data directly instead of using cleaned_data.